src/styles/_calendar.scss

```
@@ -0,0 +1,55 @@
+.calendar {
+  box-sizing: border-box;
+  padding: $padding;
+  width: ($padding * 2) + ($gap * 6) + ($size * 7);
```

what do these variables refer to?
I can't figure it out from their name.
Change the variable names
```suggestion
  width: ($.....* 2) + ($....* 6) + ($.....* 7);
```